README.md

```diff
@@ -113,6 +113,7 @@ type Metadata = {
       type: string
       width: number
       height: number
+      alt?: string
```

Comment:
I have this as an optional prop — sites are supposed to include this, but in practice totally possible that they do not.